lib/dotcom_web/components/alerts.ex

```diff
@@ -0,0 +1,36 @@
+defmodule DotcomWeb.Components.Alerts do
+  @moduledoc """
+  Function components for rendering different styles of alerts.
+  """
+
+  use DotcomWeb, :component
+
+  attr :alert, :map, required: true
+
+  @doc """
+  An alert that is embedded within another component.
+  It does not include header information like the time range, effect, or route.
+  """
+  def embedded_alert(assigns) do
+    ~H"""
+    <div class="p-4 bg-gray-100">
+      <%= if @alert.image do %>
+        <img
+          class="w-full mb-4"
+          src={@alert.image}
+          alt={if @alert.image_alternative_text, do: @alert.image_alternative_text, else: ""}
+        />
+      <% end %>
+      <p class="my-0">{@alert.header}</p>
+      <hr class="h-px my-4 bg-gray-200 border-0" />
+      <p class="my-0">{@alert.description}</p>
```

Comment:
So I'm not sure what magic we do to get this description formatting... but it'd be really nice if we could figure out how to replicate it here. Also IIRC people like seeing the "updated by" because it's reassurance that the data is fresh.  
<img width="517" alt="image" src="https://github.com/user-attachments/assets/8a92ac73-d305-46c0-8b3f-5e7ccaea1144" />

as opposed to 
<img width="581" alt="image" src="https://github.com/user-attachments/assets/46312134-281f-48b3-b4e6-b192add83cce" />

Reply:
We already found the function that does that. I'll add it in.